tools/ci_build/github/azure-pipelines/templates/jar-maven-signing-linux.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  - name: JarFileDirectory
+    type: string
+
+steps:
+  - task: AzureKeyVault@2
+    displayName: 'Get GnuPG signing keys'
+    inputs:
+      azureSubscription: 'OnnxrunTimeCodeSign_20240611'
```

Comment:
NIT: Maybe make a comment here and showing this is a Service Connection under Lotus? This way people will not go searching for an Azure Subscription with this name.

Reply:
The value for this property can only be a Service Connection Name.  We cannot directly use an Azure subscription in pipelines. We must create a service connection first.